Core/src/music/SyncChannel.cpp

```diff
@@ -5,12 +5,7 @@
 
 namespace ojgl {
 
-SyncChannel::SyncChannel()
-    : numNotes(-1)
-    , channel(-1)
-    , _minNote(-1)
-{
-}
+SyncChannel::SyncChannel() = default;
```

Comment:
This is not necessary right? already defined in the header. (same with destructor)